flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer09.java

```diff
@@ -171,6 +170,11 @@ public FlinkKafkaConsumer09(List<String> topics, KeyedDeserializationSchema<T> d
 	}
 
 	@Override
+	public FlinkKafkaConsumerBase<T> setStartFromSpecificDate(Date date) {
+		throw new RuntimeException("This method dose not support for version 0.8 of Kafka");
```

Comment:
Do you mean 0.9? Also, typo on "dose".
I would also suggest to be more specific: "Starting from a specific date is not supported for Kafka version xx".